ceph_deploy/hosts/clear/install.py

```diff
@@ -0,0 +1,14 @@
+
+def install(distro, version_kind, version, adjust_repos, **kw):
+    """
+    Install bundle that contains ceph on the clear host.
+
+    Since clear does not have alternate channels, we will just run the command
+    """
+    logger = distro.conn.logger
+    packages = ['storage-cluster']
```

Comment:
Differs from packages in uninstall.py, is this correct?

Reply:
Yes, storage-cluster is a super-set that contains ceph and other bits. the uninstall list explicitly calls ceph just to be sure that we get them all removed.